### PR TITLE
Add missing LICENSE files

### LINKED ISSUE #968: Add LICENSE files to all packages
Just noticed that not all of our packages in this monorepo have LICENSE files (even tho the license is stated in the READMEs and package.json). It would be good to include LICENSE files in each package to avoid ambiguity.

### PR DESCRIPTION
**Pull request by dapplion**
Fixes https://github.com/ChainSafe/lodestar/issues/968

I've respected the license as stated in each package, but I like to assert that the license of each package aligns with what the team wants:

| package | License |
| --- | --- |
| benchmark-utils | Apache-2.0
| lodestar | LGPL-3.0
| lodestar-beacon-state-transition | Apache-2.0
| lodestar-cli | LGPL-3.0
| lodestar-config | Apache-2.0
| lodestar-params | Apache-2.0
| lodestar-spec-test-util | Apache-2.0
| lodestar-types | Apache-2.0
| lodestar-utils | Apache-2.0
| lodestar-validator | LGPL-3.0
| spec-test-runner | LGPL-3.0

### CODE REVIEW

**Comment by mpetrunic** [comment]
@GregTheGreek I think you should take a look at this

File: packages/lodestar-beacon-state-transition/LICENSE [hunk 1]
```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```
Comment (wemeetagain) [inline packages/lodestar-beacon-state-transition/LICENSE:1]: I think this one should be apache

**Review by wemeetagain — CHANGES_REQUESTED** [review]
Thanks for tackling this

File: packages/lodestar-beacon-state-transition/LICENSE (hunk 1 above)
Comment (GregTheGreek) [reply packages/lodestar-beacon-state-transition/LICENSE:1]: I think we'll keep it how it is,,and explore dual licensing things.

**Review by wemeetagain — APPROVED** [review]

**Review by GregTheGreek — APPROVED** [review]

**Pull request merged**